### DISCUSSION TITLE
Cross-animal training

### DISCUSSION THREAD

**Discussion by zhuyuhang4** [AdaptiveMotorControlLab/CEBRA#153] [General]
I saw an example of cross-animal training on this website:https://cebra.ai/docs/demo_notebooks/Demo_hippocampus_multisession.html, where one approach involved separate training and then alignment, while the other approach trained the data together. I was wondering if it is possible to implement a large model that can infer similar manifolds for other animal individuals without further training.The number of neurons varies among different animals.

**Comment by MMathisLab** [comment]
We have an adapt function that takes in new data (animal), so check that out. 

Otherwise, we are building more universal encoder approaches, stay tuned.
